### PR TITLE
Feature/organize

### LINKED ISSUE #32: Update the doc

### LINKED ISSUE #33: Fix issue where capitalized market name isn't found in market list
We try to extract if there's a trade pair with the given command. i.e "$ETH binance USD"  or "$ETH binance usd"

Currently markets in https://github.com/YACHTPARTY/yachtbot/blob/d608e80b0ae63cf5723bdddd1f0f14baa0e9a457/src/market.js#L8 are lowercase, but given trade pair can be all in uppercase hence https://github.com/YACHTPARTY/yachtbot/blob/d608e80b0ae63cf5723bdddd1f0f14baa0e9a457/src/index.js#L52 check will fail. 

TODO: check tradepair as lowercase

### PR DESCRIPTION
**Pull request by MuhsinunCool**
Changes:
- Organized files
- Updated README.md (Potentially fixed #32?)
- Added some new replies
- Moved replies to a new file
- Fixed lots of linting errors
- Organize lots of code
- Moved ticker code to ticker.js from index.js
- Potentially fixed #33?

### CODE REVIEW

File: src/ticker.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  const symbol = symbolAndTradePairMarket[0].toUpperCase()
+
+  try {
+    if (symbolAndTradePairMarket[2]) {
+      tradePair = symbolAndTradePairMarket[1]
+      marketName = symbolAndTradePairMarket[2]
+    } else if (!symbolAndTradePairMarket[2] && symbolAndTradePairMarket[1]) {
+      tradePair = symbolAndTradePairMarket[1]
+    }
+    if (market.marketLists.hasOwnProperty(tradePair)) {
```
Comment (nazimamin) [inline src/ticker.js]: @MuhsinunCool the fix is: ```if (market.marketLists.hasOwnProperty(tradePair.toLowerCase()))``` Can you push this change?

File: src/ticker.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      marketName = tradePair
+      tradePair = null
+    }
+    // first send CMC since it's pretty instant
+    if (!marketName || marketName.toUpperCase() === 'CMC') {
+      const coinmarketcapPrice = await market.getCoinmarketcapEmbeddedContent(symbol)
+      if (coinmarketcapPrice) {
+        var response = {embed: coinmarketcapPrice}
```
Comment (nazimamin) [inline src/ticker.js]: @MuhsinunCool scoping issue with response. Your response will always return from the most outbound scope.

File: src/ticker.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      } else {
+        var response = '**$' + symbol + '** was not found on CoinMarketCap!'
+      }
+    } else if (marketName && marketName.toUpperCase() === 'BINANCE') {
+      const binancePrice = await market.getBinanceEmbeddedContent(
+        symbol,
+        tradePair
+      )
+      if (binancePrice) {
+        var response = {embed: binancePrice}
```
Comment (nazimamin) [inline src/ticker.js]: Scoping issue here as well. You will only ever need to initialize response once. And re-initialize it based on statements.

File: src/ticker.js (hunk 1 above)
Comment (MuhsinunCool) [reply src/ticker.js]: fixed

File: src/ticker.js (hunk 2 above)
Comment (MuhsinunCool) [reply src/ticker.js]: fixed

File: src/ticker.js (hunk 3 above)
Comment (MuhsinunCool) [reply src/ticker.js]: fixed

**Comment by MuhsinunCool** [comment]
Ready for PR merge :)

**Pull request merged**